seahorn/jobs/byte_buf_from_array/CMakeLists.txt

```diff
@@ -1,9 +1,12 @@
 add_executable(byte_buf_from_array
   ${AWS_C_COMMON_ROOT}/source/byte_buf.c
   aws_byte_buf_from_array_harness.c)
+target_compile_definitions(byte_buf_from_array
+  PUBLIC KLEE_MAX_SIZE=${KLEE_MAX_SIZE} # only use for compilation
```

Comment:
this should not be necessary for every benchmark. Can set globally like other definitions

Reply:
Will remove in upcoming commits.